### DISCUSSION TITLE
Settings to increase BL probing speed

### DISCUSSION THREAD

**Discussion by jjfs127** [Jyers/Marlin#58] [Q&A] [answered]
I've started from the `BL Touch 3x3 High Speed` configuration file, and notice the probe movement a little slow compared to the Smith3d firmware. Is there a particular setting(s) I could tweak for this?
Thanks

**Comment by clintiepoo** [comment]
It looks like the line that controls this is in Configuration_adv.h

```
#define BLTOUCH_HS_MODE
```

If you want to use "standard" probing, you can speed up using a couple of settings as defined [here](https://youtu.be/neS7lB7fCww?t=1048)

**Comment by Jyers** [comment, marked as answer]
XY_PROBE_SPEED is the setting you're looking for. Smith3d sets it to about 10000
